lte/gateway/c/oai/include/mme_app_ue_context.h

```diff
@@ -369,8 +360,6 @@ typedef struct ue_mm_context_s {
 
   /* TODO: Add TAI list */
   tai_t serving_cell_tai;
-  tai_list_t tail_list; // Current Tracking area list
```

Comment:
Let's keep all the TAI related context together, either all in ue_mm_context_t or all in emm_context_t.